docs/Admin.md

```diff
@@ -31,7 +31,7 @@ Here are all the props accepted by the component:
 * [`title`](#title)
 * [`dashboard`](#dashboard)
 * [`catchAll`](#catchall)
-* [`menu`](#menu)
+* [`menu`](#menu) (deprecated)
```

Comment:
Why leaving it at all in the documentation then ?

Reply:
because it's deprecated, not removed. People using this prop would be surprised not to find the related documentaiton. We'll remove it in 3.0.